src/pybind/mgr/dashboard_v2/frontend/src/app/shared/components/view-cache/view-cache.component.html

```diff
@@ -0,0 +1,14 @@
+<alert type="info"
+       *ngIf="status === vcs.ValueNone">
+  We're still trying to get your data, please wait.
```

Comment:
Not sure if "your data" is appropriate here. When exactly does this message appear? How about "Retrieving data, please wait." - the exact wording depends on the context and what expectation should be set. How much longer will this take? Can it fail, so should the message convey we're "trying"? Or does it simply take a while?

Reply:
I will change it to  "Retrieving data, please wait.".

This happens on the 1st time the user tries to get the data and most of the times, on the 2nd request (5s later) the data is already available.

There might be some endpoints that take longer than 5s.